cfme/configure/tasks.py

```diff
@@ -125,6 +131,7 @@ def check_tasks_have_no_errors(task_name, task_type, expected_num_of_tasks, sile
     # expected_status change from str to support also regular expression pattern
     expected_status = re.compile(expected_status, re.IGNORECASE)
 
+    tab_view.table.wait_displayed(fail_func=view.reload.click)
```

Comment:
**REQUIRED**
```python
wait_for(
    lambda: tab_view.table.is_displayed,
    timeout=10,
    delay=2,
    fail_func=view.reload.click
)
```

Reply:
@quarckster Yeah, that sound like much better idea. :+1:

Reply:
@quarckster Fixed